pandas/core/ops.py

```diff
@@ -1077,7 +1077,7 @@ def fill_binop(left, right, fill_value):
     return left, right
 
 
-def mask_cmp_op(x, y, op, allowed_types):
+def mask_cmp_op(x, y, op):
```

Comment:
can you add types where possible (not asking for everything of course, but if convenient / obvious)